src/create.ts

```diff
@@ -0,0 +1,14 @@
+import { AttachSink } from './types'
+import { ProxyStream } from './ProxyStream'
+import { Stream } from '@most/types'
+
+export function create<A = any>(): [AttachSink<A>, Stream<A>]
+export function create<A, B>(f: (stream: Stream<A>) => B): [AttachSink<A>, B]
```

Comment:
Is it true that `f` can return _any_ value? Does that hold up? If so, could you provide an example?

Reply:
Yes, and there is an example in the README using `hold`